BUILDING.md

```diff
@@ -283,6 +283,12 @@ may reduce build time. For more information, see the
 The above requires that `python` resolves to a supported version of
 Python. See [Prerequisites](#prerequisites).
 
+If you run into a `file not found` error, please try
```

Comment:
I'm not sure it's worth being this specific unless this is the only error seen with a stale build.

Reply:
Modified the messaging accordingly.